node/include/cocaine/detail/service/node/slave/state/handshake.hpp

```diff
@@ -0,0 +1,53 @@
+#pragma once
```

Comment:
I don't like this name change.
First of all I'll have the whole file in conflict. 
Second - state "handshaking" sound ok, state "handshake" is not anyway better.